db/traced_rows.go

```diff
@@ -0,0 +1,74 @@
+package db
```

Comment:
align the filename with the type LoggedRows <-> traced_rows

Reply:
Done!